apps/prairielearn/src/tests/fileEditor.test.ts

```diff
@@ -1133,10 +1171,34 @@ function testUploadFile(params) {
     });
   });
 
+  describe(`Uploaded file is available`, function () {
+    it('file view should match contents', async () => {
+      const res = await fetch(`${params.adminUrl}/${encodePath(params.path)}`);
+      assert.isOk(res.ok);
+      locals.$ = cheerio.load(await res.text());
+      const pre = locals.$('.card-body pre');
+      assert.lengthOf(pre, 1);
+      assert.strictEqual(pre.text(), params.contents);
+    });
+
+    it('file download should match contents', async () => {
+      const downloadUrl = locals.$('.card-header a:contains("Download")').attr('href');
+      const res = await fetch(`${siteUrl}${downloadUrl}`);
+      assert.isOk(res.ok);
+      assert.strictEqual(await res.text(), params.contents);
+    });
+  });
+
   pullAndVerifyFileInDev(params.path, params.contents);
 }
 
-function testRenameFile(params) {
+function testRenameFile(params: {
+  url: string;
+  id: string | number;
```

Comment:
Nit: did you consider dropping the `id` in favor of `path` here as well?

Reply:
Done in rename in 9ac5e513d, and in upload in c091090c0.